gson/src/test/java/com/google/gson/ParameterizedTypeFixtures.java

```diff
@@ -50,7 +51,7 @@ public String getExpectedJson() {
       return String.format("{\"value\":%s}", valueAsJson);
     }
 
-    private String getExpectedJson(Object obj) {
+    private static String getExpectedJson(Object obj) {
```

Comment:
I guess this can be static. While you are here, perhaps replace all of the `catch` clauses with a single `catch (ReflectiveOperationException e)`? (`SecurityException` is already a `RuntimeException` so it doesn't need to be caught.)

Reply:
Good point; I simplified this now also for:
- `PrimitiveTypeAdapter`
- `ProtoTypeAdapter`

Reply:
Thanks!